src/Form/Traits/AllowedLinkClassesTrait.php

```diff
@@ -93,10 +93,14 @@ public function getTypesProps(): string
             }
             $typesList[$key] = [
                 'key' => $key,
-                'title' => $type->i18n_singular_name(),
+                'title' => $type->getMenuTitle(),
                 'handlerName' => $type->LinkTypeHandlerName(),
+                'priority' => $class::config()->get('menu_priority'),
             ];
         }
+        uasort($typesList, function ($a, $b) {
+            return $a['priority'] - $b['priority'];
+        });
```

Comment:
```suggestion
        uasort($typesList, fn ($a, $b) => $a['priority'] - $b['priority']);
```
I won't block merging the PR for this - but in the future we should use the short function syntax for this sort of thing, since we're just returning a value.
The long function syntax is only really valuable when we've got multiple lines of functionality or when the return statement is really long.